ql/src/java/org/apache/hadoop/hive/ql/parse/CalcitePlanner.java

```diff
@@ -1577,7 +1577,7 @@ public class CalcitePlannerAction implements Frameworks.PlannerAction<RelNode> {
     private final Map<String, PrunedPartitionList>        partitionCache;
     private final Map<String, ColumnStatsList>            colStatsCache;
     private final ColumnAccessInfo columnAccessInfo;
-    private Map<HiveProject, Table> viewProjectToTableSchema;
+    private Map<RelNode, Table> relNodeToTable;
```

Comment:
There are a bunch of other mappings below and they all have the prefix `relToSomething` so for consistency reasons I would name this `relToTable`.

Reply:
Additionally, it seems that we could make this final and do the initialization here.